.travis.yml

```diff
@@ -5,9 +5,15 @@ python:
     - 3.4
     - 3.5
     - 3.6
-install:
+matrix:
+    include:
+        - python: "3.7"
+          dist: xenial
+          sudo: true
```

Comment:
The 'sudo' key can be dropped at both levels now, per [Travis recommendation](https://changelog.travis-ci.com/linux-builds-run-on-vms-by-default-77106).

Reply:
TIL.